patchwork/steps/CallOpenAI/CallOpenAI.py

```diff
@@ -10,9 +10,12 @@
 from patchwork.logger import logger
 from patchwork.step import Step
 
+_TOKEN_URL = "https://app.patched.codes/signin"
+_DEFAULT_PATCH_URL = "https://patch-function-ps63srnnsq-de.a.run.app"
```

Comment:
Can we avoid exposing the public cloud run URL. We can use something like "https://api.patched.codes/v1/"